File: app/src/main/java/com/siercuit/cartelera/views/FontIconView.java
package com.siercuit.cartelera.views;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.TextView;

import com.siercuit.cartelera.App;

public class FontIconView extends TextView
{

    public FontIconView(Context context) {
        super(context);
    }

    public FontIconView(Context context, AttributeSet attrs)
    {
        super(context, attrs);
        setTypeface(App.getIconsTypeface());
    }

}

File: app/src/main/kotlin/io/vextil/billboard/ui/FontIconView.kt
package io.vextil.billboard.ui

import android.content.Context
import android.graphics.Typeface
import android.util.AttributeSet
import android.widget.TextView

open class FontIconView : TextView {

    var tface: Typeface? = null

    constructor(context: Context) : super(context)

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {
        if (tface == null) {
            tface = Typeface.createFromAsset(context.assets, "fonts/icons.ttf")
        }
        typeface = tface
    }

}
